### DISCUSSION TITLE
Ideas for CSS global prayer animation

### DISCUSSION THREAD

**Discussion by squigglybob** [Pray4Movement/prayer-global-porch#28] [Ideas]
@ChrisChasm How does something like this look to you?

https://codepen.io/squigglybob/pen/rNvbYPa

[prayer animations - 18 October 2022 - Watch Video](https://www.loom.com/share/e031c6a6cfe94d05b3a3a90df2f0c532)
<a href="https://www.loom.com/share/e031c6a6cfe94d05b3a3a90df2f0c532">
    <p>prayer animations - 18 October 2022 - Watch Video</p>
    <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/e031c6a6cfe94d05b3a3a90df2f0c532-with-play.gif">
  </a>

**Comment by squigglybob** [comment]
@KK's thoughts on this

I don’t think we should try to solve the intercessor AND prayed-for polygons on the same map. My vote would be for a rotating, bouncing map that just showed places turning white (a “flash” indicating they’ve just been prayed for) with a changing metric somewhere on screen that shows key stats like maybe:
the # of total Prayer.Global intercessors ever
current prayer lap #
no. of polygons finished vs total
record lap speed
current lap pace
If the rotating, bouncing is too violent, then perhaps a simple “flash” corresponding to the completing polygon would suffice, even if it is a bit tough to discern for some parts of the map.
I’m not trying to unilaterally convince you of all this, but would value a conversation among interested parties.

**Reply by ChrisChasm** [reply]
- I know we don't want to make this a heavy loading map because it is our home page. 
- I know we don't want to distract from the call to action "Start Praying"
- I believe we are going to wait for a design review by a code company we just connected with.
